### PR TITLE
Support for java.time.Instant in Firebase Timestamp

### PR DESCRIPTION
**Pull request by cherylEnkidu**

### CODE REVIEW

**Review by dconeybe — COMMENTED** [review]
LGTM. I'll mark my review as "Approved" once the API proposal (go/java-instant-in-firebase-timestamp for Googlers) is approved by API council.

**Review by daymxn — COMMENTED** [review]
LGTM, I defer the final approval to @dconeybe for post api council.

File: firebase-firestore/CHANGELOG.md
Comment (daymxn) [inline firebase-firestore/CHANGELOG.md:3]:
```suggestion
* [feature] Support conversion between `java.time.Instant` and `Timestamp` [#5853](//github.com/firebase/firebase-android-sdk/pull/5853)
```

**Review by daymxn — APPROVED** [review]

**Review by dconeybe — APPROVED** [review]
Approved! Please wait for the base PR to be merged so you can merge this PR directly into the master branch.

**Pull request merged**
